openedx/core/djangoapps/user_api/accounts/api.py

```diff
@@ -62,7 +62,7 @@ def get_account_settings(request, username=None, configuration=None, view=None):
             "shared", only shared account information will be returned, regardless of `request.user`.
 
     Returns:
-         A dict containing account fields.
+         A dict containing account fields if only one user is requested else returns a list of dict
```

Comment:
This bubbles up to the view doesn't it? As with the profile image PR, I'm not super excited about one endpoint returning different formats depending on what parameters are passed to it.  I understand we can't change the format of the existing responses for backwards compatibility reasons.  Is it worth creating a separate endpoint for requesting multiple users @wajeeha-khalid @robrap @nasthagiri?